ipv8/attestation/trustchain/block.py

```diff
@@ -16,6 +17,7 @@
 EMPTY_SIG = b'0' * 64
 EMPTY_PK = b'0' * 74
 ANY_COUNTERPARTY_PK = EMPTY_PK
+SKIP_ATTRIBUTES = frozenset({'key', 'serializer', 'crypto', '_transaction', '_logger'})
```

Comment:
Why not just a normal `set`? `frozenset` is actually slower: https://stackoverflow.com/q/36555214 . Also, if any programmer subclasses the block, he cannot add to the `SKIP_ATTRIBUTES` set (without `setattr` magic).

Reply:
Ah. I thought mutability was not desired. Now that I know better I will change it 👍

Reply:
Done.